serialapi/request_node_info.go

```diff
@@ -4,10 +4,10 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/davecgh/go-spew/spew"
```

Comment:
Super nit: import sorting

Reply:
This is technically alphabetical. 
`github.com/davecgh`
should come before
`github.com/gozw`

Reply:
We've often followed a convention of sorting like this:
```
    import (
        "database/sql"
        "io"
        "strconv"

        "golang.org/x/net/context"

        "example.com/foo/bar"
        "example.com/foo/baz"
    )
```
- Built-ins
- 3rd party
- Local